docs/topics/downloader-middleware.rst

```diff
@@ -55,8 +55,7 @@ particular setting. See each middleware documentation for more info.
 Writing your own downloader middleware
 ======================================
 
-Each middleware component is a Python class that defines one or
-more of the following methods:
+Each middleware component is a Python class that inherits from this class:
```

Comment:
after this PR it still can be done the old way. Shouldn't that still be included in the documentation with some remark that it is outdated to do it that way?